scripts/run_mypy.py

```diff
@@ -0,0 +1,186 @@
+"""
+Invokes mypy and compare the reults with files in /pymc except tests
+and a list of files that are expected to pass without mypy errors.
+
+Exit code 0 indicates that there are no unexpected results.
+
+Usage
+-----
+python scripts/run_mypy.py [--verbose]
+"""
+import argparse
+import importlib
+import os
+import pathlib
+import subprocess
+import sys
+
+from typing import Iterator
+
+import pandas
+
+DP_ROOT = pathlib.Path(__file__).absolute().parent.parent
+PASSING = """
```

Comment:
maybe in the "unexpected_passing" message below, or as a comment here, you can mention to open an issue to delete this file once all files are passing?

Reply:
Haha, you're optimistic 😀

But yes, I'll add that next time I make a change here.